### DISCUSSION TITLE
CPU temp. and fan speed issue

### DISCUSSION THREAD

**Discussion by frozennutzz** [seerge/g-helper#3487] [Q&A]
![image](https://github.com/user-attachments/assets/9a81a747-3584-41aa-a727-d6eaa42e3291)

bro i have no fckn clue what just happen to this gem (Ghelper)
from the morning im facing multiple issues like battery % variation comparing to both the windows and what Ghelper is saying but it was later solved...

And now im facing other issues like CPU temperature not showing and my fan speed is unable to control(i am using the 'experimental Ghelper version for the only sake to control my its speed) but i tired to reinstall multiple times but its unable to control like a day ago....my experience today is literal NIGHTMARE for the past like 30 monts i was loving this...

please try to rectify these issues....this was such a lifeline for my laptop

**Comment by seerge** [comment]
@frozennutzz are you sure that you are on [Experimental build ?](https://github.com/seerge/g-helper/discussions/2272) Cause looks like you have accidentally downloaded mainstream one.

Also it could be that Asus System Control Interface is acting, [you may want to try to reinstall it](https://github.com/seerge/g-helper/wiki/Troubleshooting#reinstalling-asus-system-control-interface)

**Reply by frozennutzz** [reply]
thankyou @seerge i did deleting the existing version and its %appdata% then "Hard reset / Embedded Controller (EC) reset" as suggested in your link and idk what just happened but all the issues seemed to be fixed....

i did try to reinstall the experimental version but the same issues followed especially the CPU temp not showing issue but i deletd it and reinstalled the normal latest version but that solved the CPU temp issue not again had the fan speed issue.....

this is the experimental build im using without any issues and you make it sure it is the right one??
![image](https://github.com/user-attachments/assets/1d316308-6912-4ef9-abac-0863761984e6)

**Reply by seerge** [reply]
@frozennutzz when you use Experimental build and click `Apply Custom Fan curve` you will see actual fan speeds app tries to set in a red label under `Fans + Power`
